src/StoreKeeper/WooCommerce/B2C/Core.php

```diff
@@ -463,4 +471,93 @@ public function renderBootError(BootError $e)
             }
         }
     }
+
+    /**
+     * @param $settings
+     * @return mixed
+     */
+    public function addExtraFieldsInShippingMethods($settings): mixed {
+        // Add a new setting for Minimum Amount
+        $new_settings = $settings;
+        $currency_code = get_woocommerce_currency_symbol();
+
+        $new_settings['min_amount'] = array(
+            'title'       => __('Minimum Cost (' . $currency_code .')', 'woocommerce'),
```

Comment:
this translation does not work, please check word press documentation how to do translation when they include variables https://codex.wordpress.org/I18n_for_WordPress_Developers#Placeholders